inc/class-shortcake-bakery.php

```diff
@@ -139,6 +140,13 @@ public function action_admin_enqueue_scripts() {
 			'nonces' => array(
 				'customEmbedReverse' => wp_create_nonce( 'embed-reverse' ),
 			),
+			'shortcodes' => array_flip(
+				array_map(
+					function( $class ) {
+						return str_replace( 'Shortcake_Bakery\\Shortcodes\\', '', $class );
```

Comment:
Shouldn't we run `get_shortcode_tag()` for each class, as other namespaces won't benefit from this `str_replace()` ?